src/interface/TransactionInterface.ts

```diff
@@ -386,6 +14,9 @@ export async function processTransaction(contextEndpoint: string, transactions:
 
 	const transactionID = await fetch(transactionUrl, {
 		method: "POST",
+		headers: {
+			'Authorization': `Bearer ${keycloak.token}`
```

Comment:
@karelklima Here is an example (the token is sent for creating, pushing into, and commiting a transaction).

Reply:
Maybe seeing it from https://github.com/opendata-mvcr/ontoGrapher/pull/214/commits/208614db77dbca5a7b561feb2db4bdb6abcbecdf might make it less confusing, since only the keycloak code is added from there (the subsequent commits are minor adjustments).